src/resources/locales/en.json

```diff
@@ -477,14 +477,14 @@
         "title": "Zoom"
       },
       "preset": {
-        "apply": "Apply the preferred reading parameters",
-        "applyDetails": "Apply the preferred reading settings preset to the current book.",
-        "detail": "Save/Apply your reading presets",
-        "reset": "Reset preferred reading settings",
-        "resetDetails": "Reset the preferred reading parameters and apply these default paremeters to the current book.",
-        "save": "Save the preferred reading parameters",
-        "saveDetails": "Save the customized text parameters, so that they might be applied to other books, once open, by clicking on the button below.",
-        "title": "Preset"
+        "apply": "Apply your preferred reading parameters",
+        "applyDetails": "Apply your preferred reading parameters to the current book.",
+        "detail": "Save/Apply your preferred reading parameters.",
+        "reset": "Reset your preferred reading parameters.",
+        "resetDetails": "Reset your preferred reading parameters to their default values.",
+        "save": "Save your preferred reading parameters.",
+        "saveDetails": "ave the changes made to the current book's formatting and display choices. These settings can be applied to other books, once opened, by clicking on the button below.",
```

Comment:
@arthur-lemeur, I believe _S_ is missing at the beginning of the translation value.

Reply:
Thanks you @naglis